targets/TARGET_STM/TARGET_STM32WB/STM32Cube_FW/hw_if.h

```diff
@@ -0,0 +1,250 @@
+/* USER CODE BEGIN Header */
```

Comment:
Is this file needed in our MBED port ?

Reply:
Yes, it can be removed.
Just need to move 2 include lines.